zerver/lib/send_email.py

```diff
@@ -254,8 +254,20 @@ def send_email(
         if connection.send_messages([mail]) == 0:
             logger.error("Error sending %s email to %s", template, mail.to)
```

Comment:
Given that due to the silent failure here we don't get any useful information to log here, should we change this error message to something less similar to the others (to avoid confusion of "why is there no stack_info or other information logged with this message?)"? Eg. `logger.error("Silent failure sending %s email to %s", template, mail.to)`

Reply:
Reasonable!  Though on looking at the code:
https://github.com/django/django/blob/main/django/core/mail/backends/smtp.py#L94-L130
...I think the only way we can return 0 here, since we don't have `fail_silently` set, is if we have no recipients.  So this clause may ~never come up.

Reply:
Changed the wording.